terraform/api-specification-templates/hydrocron_aws_api.yml

```diff
@@ -248,11 +251,29 @@ paths:
               method.response.header.Access-Control-Allow-Headers: "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'"
               method.response.header.Access-Control-Allow-Methods: "'*'"
               method.response.header.Access-Control-Allow-Origin: "'*'"
+x-amazon-apigateway-api-key-source: "AUTHORIZER"
 components:
   schemas:
     json:
       type: object
     geojson:
       type: object
     csv:
-      type: object
\ No newline at end of file
+      type: object
+  securitySchemes:
+    api-key-auth:
+      type: apiKey
+      in: header       
+      name: x-api-key
```

Comment:
do these references need to change to x-hydrocron-key?

Reply:
I updated the reference for the Lambda authorizer and removed the API key. I thought this was required in order for requests to count against usage plans but after testing, I was able to confirm that requests count against the quota and once it is reached an error message is returned.